app/pages/ServiceListingPage.jsx

```diff
@@ -106,97 +106,103 @@ class ServicePage extends React.Component {
     const locations = this.getServiceLocations(service, resource, schedule);
 
     return (
-      <div className="listing-container">
-        <article className="listing" id="service">
-          <div className="listing--main">
-            <div className="listing--main--left">
-              <header>
-                <h1>{service.name}</h1>
-                {service.alsoNamed ? <p>Also Known As</p> : null}
-                <p>
-                  A service
-                  {/* TODO Implement rendering/popover when programs exist */}
-                  {program ? (
+      <div>
+        <Helmet>
+          <title>{ service.name } - AskDarcel</title>
```

Comment:
The site name itself should be parameterized based on whether we are on the whitelabeled version of the site, which uses the name "SF Service Guide". There's a helper function somewhere that abstracts that away, since it's something that depends on the domain name you view the site from.

Also, I would suggest perhaps using a vertical bar (`|`) instead of a dash here, since some resource/service names actually have dashes in them already, but it's much less likely that a vertical bar would be used in a resource/service name.